src/test/counterItem.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import LaunchService from '../modules/LaunchService.js';
+
+describe('Testing items counter', () => {
```

Comment:
- [ ] Please create a new test for the implemented counter. I'm sure you will do your best.